src/accidental.ts

```diff
@@ -297,9 +320,9 @@ export class Accidental extends Modifier {
   }
 
   // Helper function to determine whether two lines of accidentals collide vertically
-  static checkCollision(line1, line2) {
+  static checkCollision(line1: ILine, line2: ILine): boolean {
     let clearance = line2.line - line1.line;
-    let clearanceRequired = 3;
+    let clearanceRequired: number;
```

Comment:
Was the value 3 left out on purpose?

Reply:
Yes. ES-Lint made a warning that this value was not used